Source/UI/NewUI/BillingForm/Default Implementation/BillingForm/Header/Cancel/DefaultCancelButtonFormStyle.swift

```diff
@@ -13,8 +13,9 @@ public struct DefaultCancelButtonFormStyle: ElementButtonStyle {
     public var focusBorderColor: UIColor = .clear
     public var errorBorderColor: UIColor = .clear
     public var imageTintColor: UIColor = .clear
-    public var isHidden: Bool = false
-    public var isEnabled: Bool = true
+    public var isHidden = false
+    public var isEnabled = true
+    public var shouldImageFlippedForRightToLeftLayoutDirection = false
```

Comment:
Question : Why are we giving the option to merchants to set this shouldImageFlippedForRightToLeftLayoutDirection  ? I was assuming that iOS itself will take care of RTL on OS level. Especially for UI buttons as its part of the OS. ? . Please add if I missed anything.

Reply:
There's a case: 
- If the iPhone language is RTL ( Arabic ) but the the merchant's app is in LTR ( English)
then the merchant does not want button arrow image to be flipped according to Arabic RTL.
- If `ImageFlippedForRightToLeftLayoutDirection` is  forced inside the code to be always supporting **device language**  not the app language
then arrow will flip in the English app because device is Arabic.

Reply:
Yes, we can do this. I think, it's better solution.